docs/about-mono/releases/4.6.0.md

```diff
@@ -114,8 +114,30 @@ By doing so Mono runtime will not be able to output IL offsets instead of native
 
 `mono-symbolicate` can be found [here](https://github.com/mono/mono/tree/mono-4.6.0-branch/mcs/tools/mono-symbolicate)
 
+### New default GC bridge processor on Android
+
+The Android implementation makes use of a "bridge processor" which coordinates communication between the Mono and Java garbage collectors. There are three bridge processor implementations included, "old", "new" and "tarjan". In 4.6.0 the default has been changed from "new" to "tarjan". This should result in performance improvements during GC for apps that had not already selected the "tarjan" collector.
```

Comment:
Well, it has an impact if you're using java.interop, but that hasn't been released yet so I don't know what to say about that here…